sources/si-brochure/3.01/si-brochure-en.adoc

```diff
@@ -44,7 +44,7 @@ include::sections-en/99-bibliography.adoc[]
 [appendix]
 == Decisions of the CGPM and the CIPM
 
-include::sib-a1-en.adoc[leveloffset=+1,lines=21..-1]
+include::sib-a1-en.adoc[leveloffset=+1,lines=21..110]
```

Comment:
We absolutely do not like cutting a file with line numbers for many reasons. I will push a fix for this that wraps documents instead of cutting lines.